AppSettingsManagerAPI/AppSettingsManagerApi.Infrastructure/MySql/SettingGroup.cs

```diff
@@ -4,11 +4,12 @@ namespace AppSettingsManagerApi.Infrastructure.MySql;
 
 public class SettingGroup
 {
-    public string SettingGroupId { get; set; }
+    public string Id { get; set; }
     public string CreatedBy { get; set; }
 
     [Timestamp]
     public byte[] LastUpdatedAt { get; set; }
-    public List<Setting> Settings { get; set; }
-    public List<Permission> Permissions { get; set; }
+
+    public List<Setting> Settings { get; set; } = new List<Setting>();
```

Comment:
Good to make sure we can have a SettingGroup that doesn't have any associated Settings or Permissions yet